docs/en/Configuration-Settings.md

```diff
@@ -81,9 +81,27 @@ $ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
 Make sure that this file is distributed to `${ALLUXIO_HOME}/conf` on every Alluxio node (masters
 and workers) before starting the cluster.
 
-> Note that, `${ALLUXIO_HOME}/conf/alluxio-site.properties` are only loaded by Alluxio server
-> processes. Applications interacting with Alluxio service through Alluxio client will not load
-> these site property files, unless `${ALLUXIO_HOME}/conf` is on the application classpath.
+## Use Cluster Default
+
+Since v1.8, each Alluxio client can initialize the configuration with cluster-wise
+configuration values fetched from masters.
+To be specific, when different client applications like Alluxio Shell commands,
+Spark jobs, or MapReduce jobs connect to an Alluxio service,
+they will initialize their own Alluxio configuration properties with the default values
+supplied by the masters based on the `${ALLUXIO_HOME}/conf/alluxio-site.properties` files on
+masters. As a result, if masters put client-side settings (e.g., `alluxio.user.*`) or
```

Comment:
The "As a result..." sentence is a little hard to follow since it's so long - could we break to 2 or 3 shorter sentences?

Reply:
done